.travis.yml

```diff
@@ -84,8 +84,7 @@ before_install:
         contrib/travis_fastfail.sh || exit 1;
         brew tap staticfloat/julia;
```

Comment:
Can we throw a `> /dev/null` on this line while we're at it?  It spews thousands of lines of available packages.

Reply:
Done.